tests/test_hierarchy.cc

```diff
@@ -73,9 +73,10 @@ double test(std::shared_ptr<boost::property_tree::ptree> params)
       laplace._dof_handler, laplace._constraints, a, material_property);
   mfmg::Hierarchy<DVector> hierarchy(comm, evaluator, params);
 
-  pcout << "Grid complexity    : " << hierarchy.grid_complexity() << std::endl;
-  pcout << "Operator complexity: " << hierarchy.operator_complexity()
-        << std::endl;
+  // pcout << "Grid complexity    : " << hierarchy.grid_complexity() <<
+  // std::endl; pcout << "Operator complexity: " <<
+  // hierarchy.operator_complexity()
+  // << std::endl;
```

Comment:
Remove do not comment it out.

What is the rational for having/not having these print to the standard output?

Reply:
I'll remove them.

Reply:
Check with @Rombur.  I don't have an opinion on this and I am not sure why it was here in the 1st place.